src/cider/tests/nextgen/functional/FilterTest.cpp

```diff
@@ -220,123 +211,126 @@ TEST_F(CiderFilterSequenceTestNG, multiFilter) {
-  assertQueryArrow("SELECT col_3 FROM test WHERE col_3 < 50 and col_3 > 5");
-  assertQueryArrow("SELECT col_4 FROM test WHERE col_3 IS NOT NULL and col_3 > 5");
-  assertQueryArrow("SELECT col_4 FROM test WHERE col_4 < 50 and col_4 > 5");
-  assertQueryArrow("SELECT col_4 FROM test WHERE col_4 IS NOT NULL and col_4 > 5");
+  assertQuery("SELECT col_1 FROM test WHERE col_1 < 50 and col_1 > 5");
+  assertQuery("SELECT col_4 FROM test WHERE col_1 IS NOT NULL and col_1 > 5");
+  assertQuery("SELECT col_2 FROM test WHERE col_2 < 50 and col_2 > 5");
+  assertQuery("SELECT col_4 FROM test WHERE col_2 IS NOT NULL and col_2 > 5");
+  assertQuery("SELECT col_3 FROM test WHERE col_3 < 50 and col_3 > 5");
+  assertQuery("SELECT col_4 FROM test WHERE col_3 IS NOT NULL and col_3 > 5");
+  assertQuery("SELECT col_4 FROM test WHERE col_4 < 50 and col_4 > 5");
+  assertQuery("SELECT col_4 FROM test WHERE col_4 IS NOT NULL and col_4 > 5");
 }
 
 TEST_F(CiderFilterSequenceTestNG, multiColEqualTest) {
-  assertQueryArrow("SELECT col_1, col_2 FROM test WHERE col_1 = col_2");
-  assertQueryArrow("SELECT col_2, col_3 FROM test WHERE col_2 = col_3");
-  assertQueryArrow("SELECT col_2, col_4 FROM test WHERE col_2 = col_4");
-  assertQueryArrow("SELECT col_3, col_4 FROM test WHERE col_3 = col_4");
+  assertQuery("SELECT col_1, col_2 FROM test WHERE col_1 = col_2");
+  assertQuery("SELECT col_2, col_3 FROM test WHERE col_2 = col_3");
+  assertQuery("SELECT col_2, col_4 FROM test WHERE col_2 = col_4");
+  assertQuery("SELECT col_3, col_4 FROM test WHERE col_3 = col_4");
 }
 
 TEST_F(CiderFilterRandomTestNG, multiColRandomTest) {
-  assertQueryArrow("SELECT col_1, col_5 FROM test WHERE col_1 < col_5");
-  assertQueryArrow("SELECT col_2, col_6 FROM test WHERE col_2 < col_6");
-  assertQueryArrow("SELECT col_3, col_7 FROM test WHERE col_3 <= col_7");
-  assertQueryArrow("SELECT col_4, col_8 FROM test WHERE col_4 <= col_8");
-  assertQueryArrow("SELECT col_1, col_5 FROM test WHERE col_1 <> col_5");
-  assertQueryArrow(
+  GTEST_SKIP_("assertQueryIgnoreOrder method not supported yet.");
+
+  assertQuery("SELECT col_1, col_5 FROM test WHERE col_1 < col_5");
+  assertQuery("SELECT col_2, col_6 FROM test WHERE col_2 < col_6");
+  assertQuery("SELECT col_3, col_7 FROM test WHERE col_3 <= col_7");
+  assertQuery("SELECT col_4, col_8 FROM test WHERE col_4 <= col_8");
+  assertQuery("SELECT col_1, col_5 FROM test WHERE col_1 <> col_5");
+  assertQuery(
       "SELECT col_1, col_2, col_3, col_4 FROM test WHERE col_2 <= col_3 and col_2 >= "
       "col_1");
-  assertQueryArrowIgnoreOrder(
+  assertQueryIgnoreOrder(
       "SELECT col_1, col_2, col_3 FROM test WHERE col_2 >= col_3 or col_2 <= col_1", "");
-  assertQueryArrow("SELECT col_1, col_5 FROM test WHERE col_1 < col_5 AND col_5 > 0");
+  assertQuery("SELECT col_1, col_5 FROM test WHERE col_1 < col_5 AND col_5 > 0");
 }
 
 TEST_F(CiderFilterRandomTestNG, complexFilter) {
-  assertQueryArrowIgnoreOrder(
+  assertQueryIgnoreOrder(
```

Comment:
Should this `assertQueryIgnoreOrder` also be skipped? Seems handle is not consistent.